### ISSUE TITLE
Option to disable UI elements

### ISSUE THREAD

**Issue by mandreyel** [wireapp/wire-android#1433]
I would really like to have the option to disable certain buttons in chat windows. For instance, I seem to be particularly clumsy in that I regularly press on the gif/draw/voice message/timer buttons by accident. So it would be nice to have a, say, UI page in the settings menu where I'd be able to tick/untick some of these elements to hide them, as I never use them.

**Comment by LeventeLajtai** [comment]
Yes, this feature would be very useful for a more streamlined chat experience (just like turning off the emoji translation). Especially for slower phones switching to the gif/draw view can be very jarring.

**Comment by th122** [comment]
I second this -  slow phone, slow data connection, finding myself caught in wiggling molasses of giphy images  which I never want to see, let alone use makes for a rather unpleasant user experience.

It would already help to just move the most intrusive elements to the second row/screen. A less harmful element it could be exchanged for would be the attachment button, since that one only accesses local media.
